.github/workflows/rust.yml

```
@@ -0,0 +1,33 @@
+name: CI
+
+on:
```

yamllint

[warning] truthy value should be one of [false, true]

(truthy)